sample.ts

```diff
@@ -0,0 +1,3 @@
+import * as pl from "./polars"
+
+pl.col("A").str
```

Comment:
can you delete this test file. Otherwise, it looks ready to merge.

Reply:
Sorry 😬